app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,8 @@
+class FoodsController < ApplicationController
+  def index
+    @foods = FoodSearch.new.search_food(params[:q])
+  end
```

Comment:
This FoodSearch class could be turned into a facade. Currently, that `search_food` method just makes a service call via your Service file. So, you're sending parsed JSON to the view. We want to work with objects as much as possible, which is why we want your facade to iterate through the information you want to display, encapsulate them in the form of a PORO, and send them back to the view.

Reply:
Your URL has the form parameters in it because the fields from the form get sent that way, and then the index page is rendered. That's okay.